.eslintrc.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
   },
   rules: {
     '@typescript-eslint/explicit-function-return-type': 'off',
+    '@typescript-eslint/explicit-module-boundary-types': 'off',
```

Comment:
The newest version of `@typescript-eslint` adds this rule. We have this disabled in our `@bigcommerce/eslint-config` rules too.